foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/util/DataLineageUtil.java

```diff
@@ -0,0 +1,74 @@
+package com.boozallen.aissemble.upgrade.util;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class DataLineageUtil {
+
+    public static String VERSION_AISSEMBLE_POST_1_7_0 = "1.7.0";
+
+    public static final String VERSION_AISSEMBLE_POST_1_7_0_REGEX = "com\\.boozallen\\.aissemble\\.data\\.lineage";
+
+    public static final String VERSION_AISSEMBLE_PRE_1_7_0_REGEX = "com\\.boozallen\\.aiops\\.data\\.lineage";
+
+    public static final String EQUALS = "=";
+
+    public static void replaceTextInFile(File file, String textToReplace, String replacementText) {
+        Path path = file.toPath();
+
+        System.out.println("Text to Replace: " + textToReplace +"Replacement Text: " + replacementText);
```

Comment:
S: We shouldn't be printing to stdout directly.  Using a logger gives downstream consumers more control.

Reply:
Removed the function due to no usage.